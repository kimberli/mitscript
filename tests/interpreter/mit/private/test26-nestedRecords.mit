record1 = {a:5;b:6;c:{};};
record2 = record1.c;
record2.d = 8;
print("All should print 8");
print(record1.c.d);
print(record1["c"].d);
print(record1.c["d"]);
