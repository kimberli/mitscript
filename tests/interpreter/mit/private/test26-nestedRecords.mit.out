All should print 8
8
8
8
